meinberlin/react/budgeting/BudgetingProposalListItem.jsx

```diff
@@ -44,7 +44,7 @@ export const BudgetingProposalListItem = (props) => {
         numOfMoreBadges={proposal.additional_item_badges_for_list_count}
         proposalUrl={proposal.url}
       />
-      <ModeratorStatus badges={proposal.item_badges_for_list[0]} />
+      <ModeratorStatus modStatus={proposal.moderator_status} modStatusDisplay={proposal.get_moderator_status_display} />
```

Comment:
looking good, however these items don't always have a modStatus so we will need conditional formatting either here or in the sub-component to stop the code from erroring when there is no status

Reply:
it isn't shown at all if there is no mod status set, will send you a design